.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -324,6 +338,10 @@ phases:
           echo '##vso[task.setvariable variable=AZURE_VM_SIZE]Standard_DS2_v2'
           echo '##vso[task.setvariable variable=FEATURE_FLAGS]None'
         displayName: Setup Build Variables
+      - bash: |
+          echo '##vso[task.setvariable variable=DRY_RUN]True'
```

Comment:
is it possible to just set ${{ parameters.dryrun }} here?

Reply:
Looks like it can work